odoo/addons/test_lint/tests/test_checkers.py

```diff
@@ -7,7 +7,7 @@
 from subprocess import run, PIPE
 from textwrap import dedent
 
-from odoo import tools
+from odoo.tools.which import which
```

Comment:
What about moving all filesystem-related functions to a submodule like `odoo.tools.filesystem`? (Not sure about the actual name.)